examples/source/user-consent/Geolocation-based_Consent_Flow.html

```diff
@@ -75,51 +77,110 @@
+
+    If amp-geo finds that the user is in the US country group, then the contents of the US config will override the top-level consent config (and same for eu).
+
+    Our final config for a EU user would look like this: 
+    ```json
+    {
+      "consentInstanceId": "world-wide-consent",
+      "consentRequired": true,
+      "promptUI": "my-consent-ui",
+      "postPromptUI": "post-consent-ui"
+    }
+    ```
+
+    Our final config for a US user would look like this: 
+    ```json
+    {
+      "consentInstanceId": "world-wide-consent",
+      "consentRequired": "remote",
+      "checkConsentHref": "/documentation/examples/api/get-consent-sever-side",
+      "promptUI": null,
+      "postPromptUI": "post-consent-ui"
+    }
+    ```
+
+    And our final config for a user in neither of those regions would look like this: 
+    ```json
+    {
+      "consentInstanceId": "world-wide-consent",
+      "consentRequired": false,
+      "postPromptUI": "post-consent-ui"
+    }
+    ```
+
+    When a user lands on the AMP page, amp-consent will check local storage for an existing consent state. 
+    If a consent state exists, it will unblock all components. Otherwise, it will use the merged config and wait for the response from `checkConsentHref` (if set) to determine if consent is required for this user and if there is an existing consent state. If there is an existing consent state, amp-consent will use that value and unblock. Otherwise, it will show the promptUi (if configured).
+
+    Use `expireCache: true`, in your response to never save consent states to local storage and thus always prompt servers for consent state.
   -->
   <amp-consent id="myUserConsent" layout="nodisplay">
-    <script type="application/json">{
-      "consents": {
+      <script type="application/json">{
+        "consentInstanceId": "world-wide-consent",
+        "consentRequired": false,
+        "geoOverride": {
           "eu": {
-            "promptIfUnknownForGeoGroup": "eu",
-            "promptUI": "myConsentFlow"
+            "promptUI": "eu-consent-ui",
+            "consentRequired": true
+          },
+          "us": {
+            "consentRequired": "remote",
+            "checkConsentHref": "/documentation/examples/api/get-consent-sever-side",
+            "promptUI": "us-consent-ui",
+            "postPromptUI": null
           }
-      },
-      "postPromptUI": "post-consent-ui"
-    }</script>
-    <div id="myConsentFlow" class="popupOverlay">
-      <div class="consentPopup">
-        <div class="dismiss-button" role="button" tabindex="0" on="tap:myUserConsent.dismiss">X</div>
-        <h2>Headline</h2>
-        <p>This is an important message requiring you to make a choice if you're based in the EU.</p>
-        <button on="tap:myUserConsent.accept">Accept</button>
-        <button on="tap:myUserConsent.reject">Reject</button>
+        },
```

Comment:
do you think we should include an unknown group. Or is it already included in eea-preset?

Reply:
It's not in the eea-preset. So I'll add it, since it's mentioned in the spec as a best practice.